### ISSUE TITLE
detected class label count ?

### ISSUE THREAD

**Issue by srinivasb72s1** [ultralytics/yolov5#4508]
how can I count the detected labels in an image or video?

I really appreciated your work sir. please provide it sir

**Comment by glenn-jocher** [comment]
@srinivasb72s1 see PyTorch Hub tutorial:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by Guttappa1238** [comment]
@srinivasb72s1  no need to do anything, in `detect.py` 

`n = (det[:, -1] == c).sum()  # detections per class` will return count of detected objects in a frame.

**Comment by lolwakid** [comment]
> @srinivasb72s1 no need to do anything, in `detect.py`
> 
> `n = (det[:, -1] == c).sum() # detections per class` will return count of detected objects in a frame.

does "s += f"{n} {names[int(c)]}{'s' * (n > 1)}" present on line 201 affect the object count in anyway? As my produced inference video file contains the correctly classified class label along with the confidence threshold but I need the object counter for all the objects/classes. What should be done to show a counter?
